project.clj

```diff
@@ -28,27 +30,34 @@
                          [net.logstash.logback/logstash-logback-encoder "4.7"]
                          [org.codehaus.janino/janino "2.7.8"]
                          [com.fasterxml.jackson.core/jackson-core "2.7.3"]
+                         [org.yaml/snakeyaml "1.13"]
 
+                         [org.apache.maven.wagon/wagon-provider-api "2.10"]
                          [org.apache.commons/commons-exec "1.3"]
+                         [org.apache.httpcomponents/httpclient  "4.5.1"]
```

Comment:
I think this is the version we already get via `puppetlabs/http-client` version 0.5.0.  I suppose we ran into a case where some project was using the apache httpclient directly rather than puppetlabs/http-client?